docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Troubleshooting"
+description: |
+  In the following section you'll find troubleshooting guidance on source integration.
```

Comment:
A description should make sense out of context, so it doesn't work with "in the following section". (I'd also change the last word to plural.)

Reply:
Shall I get rid of the description altogether?